www/explore/teams/index.spt

```diff
@@ -0,0 +1,43 @@
+[-----------------------------]
+
+teams = website.db.all("""
+
+    SELECT t.*
+         , p.receiving
+         , p.receiving / t.nmembers AS dollars_per_member 
+      FROM ( SELECT team            AS name
+                  , count(member)   AS nmembers
+               FROM current_takes
+           GROUP BY team
+            ) AS t
+      JOIN participants p 
+        ON p.username = t.name
+  ORDER BY dollars_per_member DESC, nmembers ASC
```

Comment:
Since the focus is on receiving we should filter out the teams that don't accept tips.